src/Profiles.ts

```diff
@@ -588,14 +704,14 @@ export class Profiles {
 
         // Delete from Data Set Favorites
         const favoriteDs = datasetTree.mFavorites;
```

Comment:
I don't think favoriteDs is being used after the changes in this PR. Can we remove this declaration?